.vscode/settings.json

```diff
@@ -0,0 +1,27 @@
+{
+    "python.formatting.provider": "black",
+    "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": [
+        "--ignore=E731,W503",
+        "--max-line-length=120"
+    ],
+    "python.linting.mypyEnabled": true,
+    "python.analysis.typeCheckingMode": "off",
+    "python.linting.pydocstyleEnabled": false,
+    "python.linting.pydocstyleArgs": [
+        "--max-line-length=120"
+    ],
+    "python.linting.pycodestyleEnabled": true,
+    "python.linting.pycodestyleArgs": [
+        "--max-line-length=120"
+    ],
+    "python.linting.pylintEnabled": true,
+    "python.sortImports.args": [
+        "--profile=black"
+    ],
+    "python.testing.pytestEnabled": true,
+    "python.testing.pytestArgs": [
+        "-s",
+        "-v"
+    ]
+}
```

Comment:
I generally don't love having editor-specific configuration in version control. Can we have these line-length arguments in one of the project configuration files?

Reply:
fair enough, I'll move that into a "developer" section of the readme.

`pylint` and `mypy` are configured via `pyproject.toml` file, but `flake8, pydocstyle, pycodestyle` are not because we don't run them on github actions. Also some of them might not support `pyproject.toml` config and would require some `.files` instead.